requirements-dev.txt

```diff
@@ -1,22 +1,8 @@
-meshio >= 5.0
-networkx
-numpy
-scipy
-sympy
-matplotlib
-future
-six
-shapely
-shapely[vectorized]
+deepdiff
 pytest >= 4.6
 pytest-cov
 pytest-runner
-numba >= 0.53.0, < 0.56.1
-gmsh
 flake8
 mypy
 black == 22.8.0
```

Comment:
```suggestion
black == 22.*
```

Reply:
Starting from this year, all versions of black are stable (see https://black.readthedocs.io/en/latest/the_black_code_style/index.html#stability-policy). In short, this means that any version with a tag starting with `22` should result in the same formatted code. This motivates passing as a requirement, the less restrictive `22.*` instead of freezing a particular `22.x.x` version. 

As a bonus, this also solves  #709. Since a conda installation will currently use `22.6.0` whereas a pip installation will use `22.8.0`, although both installations will have black behaving exactly the same.

Next year we can adopt `black=23.*`, but dependabot should let us know about this automatically.

What do you think @keileg?

Reply:
I agree, that should work.

Reply:
I support adding more flexibility to formatters by adding black=23.*. It strikes a balance between strong and weak formatting constraints.